rtmp-client/src/main/cpp/librtmp/rtmp.c

```diff
@@ -338,8 +338,8 @@ RTMP_Init(RTMP *r)
   r->m_nServerBW = 2500000;
   r->m_fAudioCodecs = 3191.0;
   r->m_fVideoCodecs = 252.0;
-  //making timeout value to 10 from 30
-  r->Link.timeout = 10;
+  //Changing units to milliseconds. Changing from 10 to 10000.
```

Comment:
Remove this line (it's not a comment that makes sense when someone looks at the file - it only makes sense when they look at this change diff specifically)

Reply:
Done